common/src/main/java/net/xolt/freecam/mixins/GuiMixin.java

```diff
@@ -34,7 +27,7 @@ private void onGetCameraPlayer(CallbackInfoReturnable<Player> cir) {
     // Don't render pumpkin overlay while Freecam is active
```

Comment:
```suggestion
    // Don't render equipped-item overlays while Freecam is active
    // E.g. the pumpkin overlay
```
(if we agree this should apply to any equipped item overlay)